src/StreamJsonRpc/Protocol/JsonRpcError.cs

```diff
@@ -5,6 +5,7 @@
 using System.Runtime.Serialization;
 using StreamJsonRpc.Reflection;
 using JsonNET = Newtonsoft.Json.Linq;
+using PT = PolyType;
```

Comment:
I would avoid aliasing namespaces where they don't present type name conflicts or other confusion. The pre-existing aliases here are because two namespaces relate to JSON and present conflicting types (by name or concept), so keeping them distinct is useful. But PolyType doesn't present this problem.